G/GeoStatsImages/Compat.toml

```diff
@@ -7,5 +7,8 @@ julia = "0.5-0"
 ["0.2"]
 julia = "0.6-0"
 
-["0.3-0"]
+["0.3.0"]
 julia = "0.7-1"
+
+["0.3.1-0"]
+julia = "1.0.0-*"
```

Comment:
Keep as just 1?

Reply:
Will do in the next version. Still learning about best practices. Please merge this one for now, and I will review in the future.

Reply:
https://github.com/juliohm/GeoStatsImages.jl/pull/4